pkg/services/ngalert/models/alert_rule.go

```diff
@@ -232,6 +232,7 @@ type ListAlertRulesQuery struct {
 	OrgID         int64
 	NamespaceUIDs []string
 	ExcludeOrgs   []int64
+	RuleGroup     string
```

Comment:
I haven't made it a `*string` for the same reason `DashboardUID` is a `string` rather than `*string`. I think we can make these `*string`, but I think I would rather make all optional conditions consistent in a later PR?